packages/aragon-cli/src/commands/apm_cmds/publish.js

```diff
@@ -107,8 +107,50 @@ exports.builder = function(yargs) {
     })
 }
 
+const getMajor = version => version.split('.')[0]
+
+function getFunctionsAbi(functions, abi) {
+  functions.forEach(f => {
+    f.abi = abi.find(method => method.name === f.sig.match(/(.*)\(/)[1])
+  })
+}
+
+async function getDeprecatedFunctions(apm, artifact) {
+  let deprecated = {}
+  try {
+    let deprecatedOnVersion = []
+    const deprecatedFunctionsSig = []
+    const versions = await apm.getAllVersions(artifact.appName)
+    let lastMajor = -1
+    versions.reverse().forEach(async repo => {
```

Comment:
Can we rename `repo` to `version` (or `repoVersion`)? It's a bit weird calling `version.version` but it is what it is :).